src/pages/index.tsx

```diff
@@ -0,0 +1,611 @@
+import React, { ReactNode, useState } from "react"
+import type { GetStaticProps, InferGetStaticPropsType } from "next"
+import { useRouter } from "next/router"
+import { useTranslation } from "next-i18next"
+import { SSRConfig } from "next-i18next"
+import { serverSideTranslations } from "next-i18next/serverSideTranslations"
+import { FaGithub } from "react-icons/fa"
+import {
+  Box,
+  chakra,
+  Divider,
+  Flex,
+  FlexProps,
+  Heading,
+  HeadingProps,
+  Icon,
+  SimpleGridProps,
+  Stack,
+  useToken,
+} from "@chakra-ui/react"
+
+import { ChildOnlyProp, Lang } from "@/lib/types"
+import type { CommunityEventsReturnType } from "@/lib/interfaces"
+
+import { Image } from "@/components/Image"
+
+import { runOnlyOnce } from "@/lib/utils/cache"
+import { getLastDeployDate } from "@/lib/utils/getLastDeployDate"
+import {
+  getRequiredNamespacesForPage,
+  isLangRightToLeft,
+} from "@/lib/utils/translations"
+
+import { BASE_TIME_UNIT } from "@/lib/constants"
+
+import ActionCard from "../components/ActionCard"
+import ButtonLink from "../components/Buttons/ButtonLink"
+import CalloutBanner from "../components/CalloutBanner"
+import Codeblock from "../components/Codeblock"
+import CodeModal from "../components/CodeModal"
+import CommunityEvents from "../components/CommunityEvents"
+import HomeHero from "../components/Hero/HomeHero"
+import PageMetadata from "../components/PageMetadata"
+// TODO: migrate stats fetching on build time
+// import StatsBoxGrid from "../components/StatsBoxGrid"
```

Comment:
seems this component is not being used here

Reply:
I think this should stay in. @pettinarip mentioned this needs migration still in dev sync today